docs/extensions/extension_api.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Extension API
 category: 62c5cb137dbdad00536291a6
-hidden: true
```

Comment:
All the rest of the readme.com docs in this repo keep the attribute instead of removing it. Not sure if it matters b/c the default, when not present, might be `hidden: false`.

Reply:
Good point, just realized the same by looking at the other files, and reverted this.